lib/chef-dk/command/verify.rb

```diff
@@ -196,7 +196,7 @@ def components
 
       add_component "foodcritic" do |c|
         c.gem_base_dir = "foodcritic"
-        c.smoke_test { sh("#{embedded_bin("foodcritic -v")}") }
+        c.smoke_test { sh("#{embedded_bin("foodcritic --list")}") } # -v exits non-zero so use list
```

Comment:
The comment in the commit message about the change is probably sufficient. The code comment here mentioning `-v` would probably confuse me in a month or two

Reply:
I fixed that up